spec/student_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../student'
+
+# Test for initialize method
+describe Student do
+  it 'should create instance of the student class with 4 parameters' do
+    student = Student.new(5, 16, 'Name', 'Parent Permission')
+    expect(student).to be_an_instance_of Student
+  end
+
+  it 'should return play hooky from the play_hooky method of the Student class' do
+    student = Student.new(5, 16, 'Name', 'Parent Permission')
+    play = student.play_hooky
+    expect(play).to eq('¯(ツ)/¯')
+  end
```

Comment:
- [ ] @team great work on your tests kindly add a test for your classroom method, its a requirement to add unit tests for all methods 
![image](https://user-images.githubusercontent.com/61408860/218125222-129f4f0a-0fb7-41bd-bd16-bb6dc1375165.png)

Reply:
- Well resolved 👍🏽